lms/djangoapps/instructor_task/api.py

```diff
@@ -437,17 +444,25 @@ def submit_export_ora2_data(request, course_key):
     return submit_task(request, task_type, task_class, course_key, task_input, task_key)
 
 
-def generate_certificates_for_students(request, course_key, students=None):  # pylint: disable=invalid-name
+def generate_certificates_for_students(request, course_key, student_set=None, specific_student_id=None):  # pylint: disable=invalid-name
     """
-    Submits a task to generate certificates for given students enrolled in the course or
-    all students if argument 'students' is None
+    Submits a task to generate certificates for given students enrolled in the course.
```

Comment:
It might be useful to document the enumerated values that this method expects.  It's hard to find what the total set of values are, but this is the API, so it's probably the best place.